assignments/closure.js

```diff
@@ -1,20 +1,28 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+const phrase = "Lambda school is cool";
+function returnPhrase () {
+  return phrase;
```

Comment:
Not quite an example of a closure. The idea is that inner functions have access to variables declared in their outer functions
```
function returnPhrase() {
  const phrase = "Lambda school is cool";

  return function() {
    return phrase
  }
}
```

Here are two ways of invoking it:
```
returnPhrase.()()
```
```
const newPhraseFunc = returnPhrase()
newPhraseFunc()
```

Reply:
On my todo!